packages/lesswrong/components/languageModels/PopupLanguageModelChat.tsx

```diff
@@ -104,21 +111,11 @@ const PopupLanguageModelChat = ({onClose, classes}: {
 
   return <Paper className={classNames(classes.root, {[classes.expanded]: expanded})}>
     <AnalyticsContext pageSectionContext='llmChatPopup'>
-      <div className={classes.header}>
-        <div className={classes.title}>
-          {title}
-          <LWTooltip title="LLM chat is under development. Reviewing user conversations helps with product decisions.">
-            <div className={classes.privacyWarning}>
-              Warning! Conversation may be viewed by the LW dev team
-            </div>
-          </LWTooltip>
-        </div>
+      <div className={classes.editor}>
```

Comment:
I don't think we can get rid of the warning for all non-admin users who might still be using it